src/pages/resources/explore-books/explore-books.ts

```diff
@@ -0,0 +1,198 @@
+
+      transition('* => classAnimate', [
+        style({ width: 5, opacity: 0 }),
+        group([
+          animate('0.3s 0.2s ease', style({
+            transform: 'translateX(0) scale(1.2)', width: '*',
+          })),
+          animate('0.2s ease', style({
+            opacity: 1
+          }))
+        ])
+      ]),
+    ]),
+    trigger('ScrollHorizontal', [
+      state('true', style({
+        left: '{{left_indent}}',
+        transform: 'translateX(-100px)',
+      }), { params: { left_indent: 0 } }), // default parameters values required
+
+      transition('* => classAnimate', [
+        // style({ width: 5, transform: 'translateX(-100px)', opacity: 0 }),
+        group([
+          animate('0.3s 0.5s ease', style({
+            transform: 'translateX(-100px)'
+          })),
+          animate('0.3s ease', style({
+            opacity: 1
+          }))
+        ])
+      ]),
+    ])
+  ]
+
+})
+export class ExploreBooksPage {
+
+  categoryGradeLevels: any;
+  subjects: any;
+  mimeTypes = [
+    { name: 'ALL', selected: true, value: ['all'], iconNormal: '', iconActive: ''},
+    { name: 'TEXTBOOK', value:[], iconNormal:'./assets/imgs/book.svg', iconActive:'./assets/imgs/book-active.svg'},
+    { name: 'VIDEOS', value: ['video/mp4', 'video/x-youtube', 'video/webm'], iconNormal: './assets/imgs/Play.svg', iconActive:'./assets/imgs/Play-active.svg'},
+    { name: 'DOCS', value: ['application/pdf', 'application/epub'], iconNormal: './assets/imgs/Doc.svg',iconActive:'./assets/imgs/Doc-active.svg'},
+    { name: 'INTERACTION',
+      value: ['application/vnd.ekstep.ecml-archive', 'application/vnd.ekstep.h5p-archive', 'application/vnd.ekstep.html-archive'],
+      iconNormal: './assets/imgs/Touch.svg', iconActive: './assets/imgs/Touch-active.svg'
+    }
+  ];
+  activeMimeTypeFilter = ['all'];
+  currentFilter: string = 'ALL';
+  getGroupByPageReq: ContentSearchCriteria = {
+    searchType: SearchType.SEARCH
+  };
+  current_index: any;
+  currentGrade: any;
+  currentMedium: string;
+  layoutName = 'textbook';
+  storyAndWorksheets: Array<any>;
+  headerObservable: any;
+  unregisterBackButton: any;
+
+  constructor(
+    public navCtrl: NavController,
+    public navParams: NavParams,
+    private commonUtilService: CommonUtilService,
+    private headerService: AppHeaderService
+  ) {
+    this.handleBackButton();
+  }
+
+  ionViewDidLoad() {
+    console.log('ionViewDidLoad ExploreBooksPage');
+    this.categoryGradeLevels = this.navParams.get('categoryGradeLevels');
+    this.storyAndWorksheets = this.navParams.get('storyAndWorksheets');
+    this.subjects = this.navParams.get('subjects');
+    this.subjects.unshift({name: 'All', selected: true});
+    console.log(this.subjects);
+  }
+
+  ionViewWillEnter () {
+    this.headerObservable = this.headerService.headerEventEmitted$.subscribe(eventName => {
+      this.handleHeaderEvents(eventName);
+    });
+    this.headerService.showHeaderWithBackButton();
+  }
+
```

Comment:
No need for async on method name. you do not have await statement inside your method